Model/ShimModel.php

```diff
@@ -712,14 +717,26 @@ public function record($id, array $options = []) {
 		return $this->get($id, $options);
 	}
 
+
 	/**
-	 * Model initialization callback. To avoid overwriting constructor
+	 * Initialize a table instance. Called after the constructor.
+	 *
+	 * You can use this method to define associations, attach behaviors
+	 * define validation and do any other initialization logic you need.
 	 *
-	 * Will have param `array $config` in 3.x.
+	 * ```
+	 *  public function initialize(array $config)
+	 *  {
+	 *      $this->belongsTo('Users');
```

Comment:
These examples are not helpful, since those methods dont exist in 3.x.

Reply:
I will change to singular and backport `setPrimaryKey`.

Reply:
I wouldnt backport the new 3.6 setter ones IMO. too many new methods.
Lets keep things simple.